#! /bin/sh
patch -p0 -f $* < $0
exit $?

Fix keyserver list.


2006-02-15  Werner Koch  <wk@g10code.com>

       * Fixed typo in keyserver list.


Index: src/keyserver.c
===================================================================
--- src/keyserver.c     (revision 739)
+++ src/keyserver.c     (working copy)
@@ -169,7 +169,7 @@
       add_server (&serverlist, "hkp://random.sks.keyserver.penguin.de");
       add_server (&serverlist, "hkp://blackhole.pca.dfn.de"); 
       add_server (&serverlist, "hkp://pks.gpg.cz");
-      add_server (&serverlist, "hpk://pgp.cns.ualberta.ca");
+      add_server (&serverlist, "hkp://pgp.cns.ualberta.ca");
       add_server (&serverlist, "hkp://minsky.surfnet.nl");
       add_server (&serverlist, "hkp://keyserver.ubuntu.com");
       add_server (&serverlist, "hkp://keyserver.pramberger.at");

